src/main/java/edu/indiana/dlib/amppd/repository/CollectionSupplementRepository.java

```diff
@@ -8,5 +12,5 @@
 @CrossOrigin(origins = "*")
 @RepositoryRestResource(collectionResourceRel = "collectionSupplements", path = "collectionSupplements")
 public interface CollectionSupplementRepository extends SupplementRepository<CollectionSupplement> {
-	
+
 }
```

Comment:
Are these imports necessary? (Btw, ff you do a ctrl-shit-O Eclipse will automatically clean up unused imports.)